src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -216,6 +216,12 @@ let workbenchProperties: { [path: string]: IJSONSchema; } = {
 		'default': 'left',
 		'description': nls.localize('sideBarLocation', "Controls the location of the sidebar. It can either show on the left or right of the workbench.")
 	},
+	'workbench.panel.defaultLocation': {
+		'type': 'string',
+		'enum': ['bottom', 'right'],
+		'default': 'bottom',
+		'description': nls.localize('panelDefaultLocation', "Controls the location of the panel. It can either show at the bottom or on the right of the workbench.")
```

Comment:
I think this controls the default location, so I would prefer the folowing
`"Control the default location of the panel."